examples/elasticache/v1beta1/replicationgroup.yaml

```diff
@@ -13,7 +13,6 @@ spec:
     description: example description
     nodeType: cache.m4.large
     numCacheClusters: 2
-    parameterGroupName: default.redis7
```

Comment:
Looks like the the parameter group's version should match the engine version. Removing this so that when version 8 is available, this parameter does not break the tests.